temp-schema-generator/manifests.ts

```diff
@@ -112,9 +99,12 @@ export interface MetaTreeItemAction {
 	icon: string;
 	weight: number;
 }
-export interface MetaPropertyEditorUI extends IPrevalues {
+export interface MetaPropertyEditorUI {
+	label: string;
+	propertyEditor: string;
 	icon: string;
 	group: string;
+	config?: PropertyEditorConfig;
```

Comment:
Would it be realistic to have multiple configs to a property editor UI, e.g. having the config registered as its own extension instead of hooked on to a specific propertyEditorUI?

(Sorry if this is starting a bigger discussion)

Reply:
I thought about it, but I couldn't come up with a case where you need to extend another Property Editors config. You can't really access the config from within their editor.

In my opinion we should leave it like this for now. Then we can come back to it if a good use case shows up. (Let me know if you have one in mind :D )